src/Nethermind/Nethermind.Optimism/OptimismSynchronizerModule.cs

```diff
@@ -27,7 +27,7 @@ protected override void Load(ContainerBuilder builder)
             builder.AddSingleton<ITotalDifficultyStrategy>(
                 new FixedTotalDifficultyStrategy(
                     new CumulativeTotalDifficultyStrategy(),
-                    fixesBlockNumber: parameters.BedrockBlockNumber - 1,
+                    fixesBlockNumber: parameters.BedrockBlockNumber!.Value - 1,
```

Comment:
if it is required maybe it shouldn't be nullable?

Reply:
I think it's good to make everything nullable. So it will not set zero by default and we will not forget to set it when it's needed

Reply:
ok but handle `null` in elegant way then

Reply:
Can we use `TypeDiscovery` class for that?